feature/feature.go

```diff
@@ -70,7 +70,7 @@ func init() {
 // To add a new feature, define a key for it above and add it here.
 var defaultClusterAPIFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	// Every feature should be initiated here:
-	MachinePool:                    {Default: false, PreRelease: featuregate.Alpha},
+	MachinePool:                    {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
I have no clue, but wondering if it should be changed to Beta

Reply:
+1 in core k8s features are defaulted to true in beta.

Reply:
Sounds good to me. After a few years it's fair to call it beta :)